amundsen_application/static/js/components/ProfilePage/index.spec.tsx

```diff
@@ -9,7 +9,7 @@ import Breadcrumb from 'components/common/Breadcrumb';
 import Flag from 'components/common/Flag';
```

Comment:
I really prefer this way of keeping the tests beside the tested components.
I assume we plan to move the rest of tests to this place, right?

Reply:
Yes.